### DISCUSSION TITLE
Hoping to make one...

### DISCUSSION THREAD

**Discussion by K9SWR** [Short-bus/pilomar#49] [General]
I have my stepper motors, lazy susan bearing, worm and spur gear and of course an Rpi. The bearings are on the way. And I have an order into Adafruit for the lens & camera (and a couple of extra things of course).
I'm excited to try this. Hoping I can make it work.
I did have one question. I've note for clocks and other precesion type movements it has been suggested to clean out the heavey grease in the bearings and replace it with something lighter. Although, it looks like you have plenty of stepper motor to drive the mechanism.
Dan

**Comment by Short-bus** [comment]
Sounds like you've got the key components inbound. I'm sure you'll get it running, it's worked quite reliably for me so far and the image stacking software pulls interesting detail from even this modest setup.

Interesting question about the grease in the bearings, actually I didn't need to do anything. I used budget bearings bought online from Radio Controlled Car suppliers. You can get the same at higher quality & price from engineering sources, but I don't think it's needed. The Lazy Susan bearings I had were slightly 'rough running' if they sagged under weight, but those little load bearings in the foundation solved that nicely. After that it all worked quite cleanly.

You're right about the motor strength, I used 2A motors, and only had to give them about 0.7A in practice - they have plenty of capacity.
I tried smaller motors (1.2A) but they were not as smooth, the 2A ones seem to hit a sweet spot.

If anywhere has the potential for some dry friction it may be those worm gears but you've a lot of strength from those motors and it's 60:1 gearing! I initially put some light oil on the worm gears and some spray grease into the Lazy Susan on my first build, it just helped the dust to stick. The second build was left dry and has worked nicely.

The 3D printed gear parts have enough adjustment in them that you should get them to run smoothly regardless of printing/fit tolerances.

I recommend to start with the 16mm lens - it has a very wide field of view, so things are relatively small, but it is easier to get everything tuned and running. I'm now on an adapted 50mm SLR lens for this winter if the sky ever appears. That's more finicky to use, but works. I'm getting no images at all at the moment due to constantly cloudy nights.

It will be interesting to see how different observing conditions around the world impact the tuning required. I am currently tweaking the software to help with problem solving as issues crop up for people. I'll post on Instructables and GitHub as I publish updates.

Good luck with the build! There are quite a few underway at the moment, I'm waiting to see who starts getting the first images. 
I'm expecting the questions to switch from 'build' to 'run' soon.

Matt

**Comment by K9SWR** [comment]
I just got the camera and the lens (the one you suggested - seems like it would the best overall for what I'm wanting anyway).
I got the bearings and driver boards. 
I was thinking of getting these to help sort out wiring for the drivers:
[Amazon Link](https://www.amazon.com/DRV8825-Stepper-Expansion-Printer-Control/dp/B08RP2SCJ7/ref=pd_ybh_a_sccl_19/131-1909983-9095339?pd_rd_w=WtYNn&content-id=amzn1.sym.67f8cf21-ade4-4299-b433-69e404eeecf1&pf_rd_p=67f8cf21-ade4-4299-b433-69e404eeecf1&pf_rd_r=Q2QWN2PQVVEJP3QR03A3&pd_rd_wg=l6Pup&pd_rd_r=3baf1e6d-4f2d-4efe-96ff-ac8168b64126&pd_rd_i=B08RP2SCJ7&th=1)

**Reply by Short-bus** [reply]
Hi, yes, those little boards will work, I used some of those early on too.
The microstepping mode is done via the three dip switches rather than using the signal from the Tiny2040, but that's not a problem, by default the telescope always uses full steps, so you can leave the three dip switches set to 'OFF' for that if I remember correctly.
![steppercarrier](https://github.com/Short-bus/pilomar/assets/84185921/2af51d80-285b-4f1c-b719-4af8b3ee0314)
